pkg/pillar/cmd/nim/nim.go

```diff
@@ -554,9 +562,27 @@ func (n *nim) initSubscriptions() (err error) {
 	}
 
 	// We get DevicePortConfig from three sources in this priority:
+	// 0. A request from monitor TUI application (manual override)
 	// 1. zedagent publishing DevicePortConfig
 	// 2. override file in /run/global/DevicePortConfig/*.json
 	// 3. "lastresort" derived from the set of network interfaces
+	n.subDevicePortConfigM, err = n.PubSub.NewSubscription(pubsub.SubscriptionOptions{
+		AgentName:   "monitor",
+		MyAgentName: agentName,
+		TopicImpl:   types.DevicePortConfig{},
+		Persistent:  true,
```

Comment:
This most likely does not need to be persistent since we persist currently known DPCs using `pubDevicePortConfigList` (i.e. in `/persist/status/nim/DevicePortConfigList`) - I assume this includes the manual DPC.

Reply:
@milan-zededa yes, agree. I had this set to false for experements, but this change fall through the cracks.